docs/integration/linux-integration.md

```diff
@@ -0,0 +1,63 @@
+title: Linux Logs Integration
+description: Sematext Linux Logs integration allows you to send systemd's journald logs to Sematext via Logagent over HTTPS.
+
+You can send Linux journal logs to Sematext through Logagent. Once they're in, you can explore them and drill into specifics, such as SSH or Kernel logs.
+
+<img
+  class="content-modal-image"
+  alt="Linux Logs Overview"
+  src="../../images/agents/linux_overview.png"
+  title="Linux Logs Overview"
+/>
+
+## Linux Logs Quick Start
+
+With [Node.js installed](https://nodejs.org/en/download/package-manager/), you'd first need to install Logagent:
+```bash
+sudo npm i -g @sematext/logagent
+```
+
+Then configure it to push logs to your Logs App:
+```bash
+sudo logagent-setup -i $YOUR_LOGS_TOKEN -u https://logs-token-receiver.sematext.com -j
+```
+Note: if you're in the EU regions, use `-u https://logs-token-receiver.eu.sematext.com` instead.
+
+Finally, start Logagent:
+```bash
+sudo systemctl start logagent
+```
+
+## Troubleshooting
+
+If logs don't come in, it might be due to the last cursor position being >10MB away. If you truncate the journal to less than 10MB, this will be fixed:
+
+```bash
+journalctl --vacuum-size=9M
+systemctl restart systemd-journal-upload
+```
+
+For more details, have a look at our <a href="../journald-integration" target="_blank" rel="noopener">Journald integration</a>. There, you can find instructions on how to send journald entries to Sematext Logs without needing a local Logagent. That said, [in most systemd versions](https://github.com/systemd/systemd/pull/15347) you need a local Logagent for an HTTPS connection instead of plain HTTP.
+
+## Exploring logs
+
+Once data is in, you can explore it using the built-in reports or create your own. For example, you can use the Auth report to explore sudo commands:
+
+<img
+  class="content-modal-image"
+  alt="Linux Logs Auth Report"
+  src="../../images/agents/linux_auth_logs.png"
+  title="Linux Logs Auth Report"
+/>
+
+Other built-in reports include:
+
+- **Kernel**: Logs filtered by the facility 0 (kernel). You will find here all your startup logs, information about crashes, all that you typically see via `dmesg`
```

Comment:
You will find here all your startup logs
=>
Here you will find all your startup logs